pkg_defs/sherpa/build.sh

```diff
@@ -0,0 +1,16 @@
+# if [ "$(uname -s)" == "Darwin" ]
+# then
+#   CONDA_BUILD_SYSROOT=/opt/MacOSX11.0.sdk
+# fi
+
+# #Apply patches
+# git apply -v --whitespace=nowarn ${RECIPE_DIR}/patches/*.patch
+# #Ignore the changes when setting the version
+# # We only ignore files that were CHANGED by the patches, not any CREATED by the patches
+# for CIAO_IGNORE in "setup.cfg conda_build_config.yaml"
+# do
+#   git update-index --assume-unchanged ${CIAO_IGNORE}
+# done
```

Comment:
Most of this is commented out, so is it just here for clues if there are issues in the future?

Reply:
There are also the old notes in the archive sherpa recipe.